File: PRAKTIKUM MODUL 6/PRAK601-2410817310009-Muhammad_Ghazi_Rakhmadi.py
baris, kolom = map(int, input().split())
angka=[]
x = 0
isi = list(map(int, input().split()))

for i in range (baris):
    angka.append(isi[x:x+kolom])
    x = x + kolom
for baris in angka:
    for isi in baris:
        print(isi, end=' ')
    print()

File: PRAKTIKUM MODUL 6/PRAK601-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main(){
  int baris, kolom, isi, g, z;
  scanf("%d %d", &baris, &kolom);
  
    int matrix[baris][kolom];
    for (g = 0; g < baris; g++) {
      for (z = 0; z < kolom; z++) {
        scanf("%d", &isi);
        matrix[g][z] = isi;
      }
    }      
    for (g = 0; g < baris; g++) {
      for (z = 0; z < kolom; z++) {
        printf("%d ", matrix[g][z]);
      } printf("\n");
    }
}